FINS/iocBoot/iocFINS-IOC-01/config.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" ?>
 <ioc_config xmlns:xi="http://www.w3.org/2001/XInclude">
   <config_part>
-    <ioc_desc>Omron PLCs</ioc_desc>
+    <ioc_desc>Omron PLC</ioc_desc>
     <ioc_details>Driver to talk to fins activated from settings.</ioc_details>
     <macros>
-      <macro name="PLCIP" pattern="^[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+$" description="IP address of PLC" hasDefault="UNKNOWN" />
-      <macro name="PLCNODE" pattern="^$|^([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])$" description="Node of PLC" defaultValue="" hasDefault="YES" />
-  </macros>
+      <macro name="PLC_IP" pattern="^[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+$" description="IP address of PLC" hasDefault="UNKNOWN" />
```

Comment:
This will break any configurations that currently use the FINS ioc. Please revert

Reply:
Done